resources/styles/molecules/tabs/BpTabs.vue

```diff
@@ -96,117 +100,143 @@ export default {
-    },
+    })
 
-    data () {
-        return {
-            activeTab: '',
-            currentFocusTab: '',
-        }
-    },
+    //const { isActive } = useKeyboard(props.tabs, activeTab)
 
-    computed: {
-        tabKeys () {
-            return Object.keys(this.tabs)
-        },
-        firstTabKey () {
-            return this.tabKeys[0]
-        },
-        lastTabKey () {
-            return this.tabKeys[this.tabKeys.length - 1]
-        },
-        currentFocusTabIndex () {
-            return this.tabKeys.indexOf(this.currentFocusTab)
-        },
-        nextTabKey () {
-            return this.currentFocusTab === this.lastTabKey
-                ? (this.wrap ? this.firstTabKey : null)
-                : this.tabKeys[this.currentFocusTabIndex + 1]
-        },
-        prevTabKey () {
-            return this.currentFocusTabIndex === 0
-                ? (this.wrap ? this.lastTabKey : null)
-                : this.tabKeys[this.currentFocusTabIndex - 1]
-        },
-    },
-
-    mounted () {
-        this.setInitFocusTab()
-        this.setInitActiveTab()
-        this.attachKeyboardListeners()
-    },
-
-    methods: {
-        setInitActiveTab () {
-            if (location.hash && this.tabKeys.indexOf(location.hash.substr(1)) !== -1) {
-                this.activeTab = location.hash.substr(1)
-            } else if (this.initialTab) {
-                this.activeTab = this.initialTab
+    const initialTabComp = computed({
+        get: () => {
+            if (!props.initialTab) {
+                var [firstTab] = Object.keys(props.tabs)
+                return firstTab
             } else {
-                [this.activeTab] = this.tabs
+                return props.initialTab
             }
-        },
-        setFocusTab (tab) {
-            this.currentFocusTab = tab
-            this.selectTab(tab)
-        },
-        setInitFocusTab () {
-            this.currentFocusTab = this.initialTab
-        },
-        selectTab (tab) {
-            this.activeTab = tab
-            if (this.setHash) {
-                history.replaceState(undefined, undefined, `#${this.activeTab}`)
-            }
-        },
-        isActive (tab) {
-            return tab === this.activeTab ? this.activeClass : ''
-        },
-        isActiveTab (tab) {
-            return this.activeTab === tab
-        },
-        isFocusedTab (tab) {
-            return this.currentFocusTab === tab
-        },
-        navigateNextTab (evt) {
-            if (this.nextTabKey) {
-                evt.preventDefault();
-                const nextTabButton = this.$el.querySelector(`[id="${this.nextTabKey}"]`)
-                nextTabButton.focus()
-            }
-        },
-        navigatePrevTab (evt) {
-            if (this.prevTabKey) {
-                evt.preventDefault();
-                const prevTabButton = this.$el.querySelector(`[id="${this.prevTabKey}"]`)
-                prevTabButton.focus()
+        }
+    })
+
+    const wrapComp = computed({
```

Comment:
I'm not sure you need this computed property. Instead you could set a default state to the `wrap` prop like so: 

`wrap: {
            type: Boolean,
            default(props) {
                return !props.vertical;
            },
        }`

Also, just a note on Vue properties: best practice is that they are always either a) required or b) have a default value. Since `wrap` is not required, it should have a default value regardless.